### DISCUSSION TITLE
Stabilization cone (and settings) only active when Purge in prime tower checked?

### DISCUSSION THREAD

**Discussion by mrhasbean** [OrcaSlicer/OrcaSlicer#5292] [Q&A]
I have both IDEX and 2-in-1-out printers. I use a small prime tower on the IDEX to do what it is designed to do,  but am having issues with the occasional ooze blob catching on the nozzle, causing the tower to dislodge. 

On the 2-in-1-out this is overcome by using the stabilization cone to support the tower. But it seems the cone and its settings are only active if the tower is also used for purging {Purge in Prime Tower checked in the printer settings), not if just for priming? 

Before I post it as a bug or feature request I was wondering if anyone knows if this is by design, or is it a bug, and if by design, why?

**Comment by noadgskill** [comment]
I too have the same problem. Even if you activate the Prime Tower, it does not appear immediately. It is very inconvenient to have the Purge Tower activated to create the Prime Tower. An independent prime tower option is required.

Prime Tower is not necessarily used only for purging. For prints that do not have enough cooling time, such as small cone shapes, the fuzzy tower is an important element in providing cooling time for prints in small areas more quickly and stably than a draft shield. I hope this feature is restored quickly.
